### DISCUSSION TITLE
[2109460]乡村狂想曲

### DISCUSSION THREAD

**Discussion by DMWJKWDA** [pjy612/SteamManifestCache#684] [Share]
### Steam 游戏ID

2109460

### 密钥列表

```hash
2109461;f77ae6d25c175811951ff2fb68f85fb653d802e950393094175b1f84598999a1
```

### 清单附件

[2109460.zip](https://github.com/user-attachments/files/15522323/2109460.zip)

### 备注

没有内测码
邮箱：vkgjflh8384358@gmail.com

**Comment by DMWJKWDA** [comment]
ovo

**Comment by pjy612** [comment]
* 密钥格式错误 #26

**Reply by DMWJKWDA** [reply]
OvO

**Reply by pjy612** [reply]
游戏id 却又错了。。。

**Reply by DMWJKWDA** [reply]
ovo

**Reply by DMWJKWDA** [reply]
大佬给个那个软件那个赞助了之后的那个软件电脑重装没了

**Reply by pjy612** [reply]
哪个软件赞助？
这个项目可一直都没有赞助
当然 首页那个 打赏 也不算是 赞助这个项目
